App.tsx

```diff
@@ -1,11 +1,23 @@
 import { StatusBar } from "expo-status-bar";
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
+import { ScrollView, StyleSheet, View } from "react-native";
+import Story from "./components/story/Story";
 
 const App = (): JSX.Element => {
   return (
     <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
+      <ScrollView
+        showsHorizontalScrollIndicator={false}
+        horizontal
+        style={{ flexDirection: "row", marginTop: 50 }}
```

Comment:
styles should always be defined within `StyleSheet.create`. This will allow you to use a reference system designed for it instead of creating a new object on every render

Reply:
There is also an eslint rule for that
https://github.com/Intellicode/eslint-plugin-react-native/blob/master/docs/rules/no-inline-styles.md